backstory/src/main/java/com/google/sps/perspective/PerspectiveStoryAnalysisManager.java

```diff
@@ -110,22 +110,14 @@ public StoryDecision generateDecision(String story) throws NoAppropriateStoryExc
    * as a PerspectiveDecision object (for demo purposes).
    *
    * @param story The story to be analyzed
-   * @return An object describing the recommendation resulting from the analysis.
-   * @throws NoAppropriateStoryException if story is not considered appropriate
+   * @return An object describing the recommendation resulting from the analysis. If the
+   *    the PerspectiveDecision story field is null, it's not an appropriate story.
    */
-  public PerspectiveDecision generatePerspectiveDecision(String story)
-      throws NoAppropriateStoryException {
+  public PerspectiveDecision generatePerspectiveDecision(String story) {
     PerspectiveAPIClient apiClient = new PerspectiveAPIClient(perspectiveAPI);
     PerspectiveValues storyValues = apiClient.analyze(Arrays.asList(REQUESTED_ATTRIBUTES), story);
     boolean isStoryAppropriate = ContentDecisions.makeDecision(storyValues);
 
-    // if content decisions returns that it's appropriate
-    // then return a StoryDecision object with this story
-    if (isStoryAppropriate) {
-      return new PerspectiveDecision(story, storyValues);
-    }
-
-    // otherwise throw the NoAppropriateStoryException
-    throw new NoAppropriateStoryException("The story passed in was not appropriate.");
+    return new PerspectiveDecision(story, isStoryAppropriate, storyValues);
   }
```

Comment:
No tests needed to be updated?

Reply:
This is only changing elements used in the demo, which we decided not to test.

Reply:
Done

Reply:
I think that's fine for now... but I should probably clarify that when deciding what to test, it would be safer to make that distinction based on where the code lives. If it lives in /demo then it's safe to avoid testing, if it lives in /src/main, then there's nothing stopping people from using this method once it's added to the public API. In general we consider the public API part of the true functionality of a class, even if currently it is exposed just for tests/demos.

Reply:
Ok. That makes sense. I'll keep that in mind for the future.

Reply:
Done